rpmbuild/main.py

```diff
@@ -256,6 +258,11 @@ def build_rpm(args, config):
     task = get_task(args, config, build_config_url_path, task_id)
     log_task(task)
 
+    copr_rpmbuild_config = Config()
+    copr_rpmbuild_config.load_config()
+    if subscription_required(task, copr_rpmbuild_config):
+        wait_for_subscription()
```

Comment:
One negative of waiting here directly is that this task is already `running` -> so on Frontend, it seems we are actually already doing the build (not waiting for something); staying in "starting" would be nicer at least.